2주차/[Lv.2]_다리를_지나는_트럭/조은서.js

```diff
@@ -0,0 +1,23 @@
+function solution(bridge_length, weight, truck_weights) {
+  let answer = 0;
+  let arr = Array.from({ length: bridge_length }, () => 0);
```

Comment:
```suggestion
  const arr = Array.from({ length: bridge_length }, () => 0);
```
변수 `let`보다는 상수 `const`가 좋을 것 같습니다!!! 🙋🏻‍♂️